ome/omero-iviewer
# Apps Name

continuation of https://github.com/ome/omero-iviewer/pull/14

TEST: 
- check that the application now 'lives' under /iviewer (browser address bar)
- open the browser dev tools, reload and check that there are no requests failing
